SEFramework/src/lib/Frame/Frame.cpp

```diff
@@ -188,7 +188,14 @@ void Frame<T>::setVarianceMap(std::shared_ptr<WeightImage> variance_map) {
 
 template<typename T>
 void Frame<T>::setVarianceThreshold(WeightImage::PixelType threshold) {
-  m_variance_threshold = threshold;
+
+  // set the variance threshold if it make sense
+  if (m_variance_threshold < std::numeric_limits<WeightImage::PixelType>::max()){
```

Comment:
Should we warn?

Reply:
`std::numeric_limits<WeightImage::PixelType>::max()` is the default value which is taken when the user give **no** value. That line only prevents the value from spilling over to negative or similar and does **not** overwrite user input.

So no need for a warning from my side.